src/wp-admin/options.php

```diff
@@ -179,8 +191,6 @@
 
 	$allowed_options['general'][] = 'users_can_register';
 	$allowed_options['general'][] = 'default_role';
-
```

Comment:
One nit-pick -- I'd leave this blank line between general and writing.

Reply:
@ironprogrammer Thanks for the feedback, I have implemented the suggestions.